### ISSUE TITLE
make netctl cli consistent with `docker network`

### ISSUE THREAD

**Issue by jainvipin** [contiv/netplugin#239]
It is often difficult to remember and use various options in `netctl net` command options. Would it make sense to keep ti consistent to `docker network` cli? I am opening the thread to discuss this:

Consider the two:

######  Docker Network Options
```
Usage:	docker network [OPTIONS] COMMAND [OPTIONS]

Commands:
  create                   Create a network
  connect                  Connect container to a network
  disconnect               Disconnect container from a network
  inspect                  Display detailed network information
  ls                       List all networks
  rm                       Remove a network
```

###### Whereas `netctl net` output is:

```
NAME:
   netctl net - Network manipulation tools

USAGE:
   netctl net command [command options] [arguments...]

COMMANDS:
   list		List networks
   delete	Delete a network
   create	Create a network
   help, h	Shows a list of commands or help for one command
   
OPTIONS:
   --help, -h	show help
```

It seems it would be good to change netctl commands to map
- list --> ls
- delete --> rm
- create --> create (same)
- inspect --> should we implement (along the lines of `docker network inspect`

Of course, this means we change the verbs for all commands, then - like `policy`, `tenant`, etc.

@unclejack, @erikh, @mapuri, @shaleman, @jojimt - WDYT?

**Comment by vijayvikrant** [comment]
@jainvipin +1 for consistency..
And while we are at it, why not netctl net ==> netctl network ..we should support both.

**Comment by jainvipin** [comment]
agree @vijayvikrant `network` sounds good; in fact with autocompletion, I don't think even `net` would work just as easily.

**Comment by jainvipin** [comment]
commenting on my own comment:

> inspect --> should we implement (along the lines of docker network inspect

While it may sound that `docker network inspect` is showing all the details, why put something in parallel that does the same. My point was to show other details, like epgs associated with the network, network's tenancy, and of course the network details, any runtime information like, ipam allocation pool remaining, any stats, etc. Perhaps a separate PR if we pick this up at all.
